lua/textcase/plugin/plugin.lua

```diff
@@ -84,13 +83,7 @@ function M.register_keys(prefix, method_table, keybindings)
   })
 end
 
-function M.register_replace_command(command, method_keys)
```

Comment:
It was part of a bigger plan. I introduced that dict as part of [shake.nvim](https://github.com/johmsalas/shake.nvim) a plugin for registering transformations and make them available to Telescope, etc, etc... at some point I decided to reduce the scope to cover only text cases and later extract all the core to shake.nvim

I don't really know if that is the direction we want to take, wdyt?

Reply:
But this would still be possible via the `methods_by_method_name` state. But maybe I didn't get what you're aiming at.

Reply:
True, and it is not worth to keep tech debt for future unplanned features. If this comes in handy we can always cherry pick it